controller/kv/helper.go

```diff
@@ -1509,7 +1564,10 @@ func (m clusterHelper) GetVulnerabilityProfile(name string, acc *access.AccessCo
 
 func (m clusterHelper) PutVulnerabilityProfile(cp *share.CLUSVulnerabilityProfile, rev *uint64) error {
 	key := share.CLUSVulnerabilityProfileKey(cp.Name)
-	value, _ := json.Marshal(cp)
+	value, err := json.Marshal(cp)
+	if err != nil {
```

Comment:
// not for PutVulnerabilityProfile()
The returned error in line 1580
```
func (m clusterHelper) PutVulnerabilityProfileIfNotExist(cp *share.CLUSVulnerabilityProfile) error {
	key := share.CLUSVulnerabilityProfileKey(cp.Name)
	value, _ := json.Marshal(cp)    //<----
	return cluster.PutIfNotExist(key, value, false)
}
```
is not handled like in PutVulnerabilityProfile()

Reply:
fixed.